tests/unit/testResources/schema.ts

```diff
@@ -15,7 +16,7 @@ export const TEST_CCLOUD_SCHEMA = Schema.create({
 
 export const TEST_LOCAL_SCHEMA = Schema.create({
   id: "1",
-  subject: "test-topic-value",
+  subject: `${TEST_LOCAL_KAFKA_TOPIC.name}-value`,
```

Comment:
Adjusting test setup assumptions so `TEST_*_SCHEMA` will always match `TEST_*_KAFKA_TOPIC`'s name via `TopicNameStrategy` and require less manually-associating in the tests themselves.